### ISSUE TITLE
Stat description format '$+d%' is not handled correctly

### ISSUE THREAD

**Issue by brather1ng** [OmegaK2/PyPoE#20]
Translations miss the leading '+' for the format '$+d%':

```python
from PyPoE.poe.file import GGPKFile
from PyPoE.poe.file import TranslationFileCache
ggpk = GGPKFile()
ggpk.read('D:/Program Files (x86)/Grinding Gear Games/Path of Exile/Content.ggpk')
ggpk.directory_build()
tf = TranslationFileCache(path_or_ggpk=ggpk)['skill_stat_descriptions.txt']
tr = tf.get_translation(
    tags=['supported_active_skill_gem_quality_%'],
    values=[20]
)
print("Expected: +20% to Quality of Supported Active Skill Gems")
print("Actual:   %s" % tr[0])  # '20% to Quality of Supported Active Skill Gems'
```

This is correct in the wiki (see e.g. http://pathofexile.gamepedia.com/Enhance_Support), so maybe I'm doing something wrong or the bug is new.

**Comment by OmegaK2** [comment]
Huh that's odd, probably broke something earlier when I fixed #18, I have a test covering the $+d case (which works) but not for the $+d%% case, which seems to be broken. Looking into it.

**Comment by brather1ng** [comment]
Seems like that change moved the '%' into the format for '$+d' (now '$+d%'). See [my stat_translations diff](https://github.com/brather1ng/RePoE/commit/778986ab5a87da69c0b20865447b9e802a45f9a1#diff-1666b5eb664bdc01f2eebe0aba0c8f62). That leads to the '+' not being prepended in `TranslationString.format_string()`.

Somewhat related: That method uses a '%d' format, so the rare cases of floating point numbers in stats are translated incorrectly:

```python
tf.get_translation(tags=['additional_base_critical_strike_chance'], values=[110])
# 1% to Critical Strike Chance
```

**Comment by OmegaK2** [comment]
I must have tested that wrong somewhere, I specifically overrode the $+d format to return digits, not floating points, must have messed up in between python formatting and the format GGG uses. Both of these issues will be fixed soon along with #19.

**Comment by brather1ng** [comment]
All looks good now, thanks!
